src/upload/catalog/controller/extension/payment/libwebpay_rest/HealthCheck.php

```diff
@@ -120,7 +158,11 @@ private function getServerResume(): array {
         ];
     }
 
-    // crea array con la informacion de comercio para posteriormente exportarla via json
+   /**
+    * Creates an array with commerce information to be exported via JSON.
```

Comment:
I think indicating that it is for exporting to JSON is unnecessary, since the method only returns the array and has no impact on what will be done with the result later.